packages/testlab/package.json

```diff
@@ -22,5 +22,13 @@
     "should": "^11.2.1",
     "sinon": "^1.17.7",
     "supertest": "^3.0.0"
-  }
+  },
+  "files": [
+    "README.md",
+    "index.js",
+    "index.d.ts",
+    "lib",
+    "lib.es2015",
```

Comment:
This says `lib.es2015`, but #225 produces `lib-es2015` AFAICT.

TBH, I would personally use `lib6`, because it shorter and the suffix `6` matches both ES6 and Node 6.